### PR TITLE
Add overview docs for new kingfisher platform

### PR DESCRIPTION
**Pull request by robredpath**

### CODE REVIEW

**Comment by robredpath** [comment]
@odscjames this is what I'm thinking for the kind of docs that will live in this repo. Crucially, this has gotten me thinking about what other resources we might need, as I wrote the 'step-by-step' section and asked myself "if I didn't already know how to do this, where would I find out?"

File: docs/hosted.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+=================
+
+Open Contracting Partnership operates a hosted instance of the Kingfisher tool suite for the use of OCP staff and OCDS Team members. If you think you should have access to this, contact `<mailto:code@opendataservices.coop>`_.
+
+Access
+------
+
+The server can be found at 195.201.163.242. You can connect via SSH to run commands, or via a Postgres client to run queries on the database (see below)
+
+There is a development server at ocdskingfisher-dev.default.opendataservices.uk0.bigv.io. If you're involved in the development of new Kingfisher components, you may be asked to log into the dev server to try things out. If so, substitiute the address of the dev server for the address of the live server in the examples below.
```
Comment (odscjames) [inline docs/hosted.rst]: Technically this does not cover all the details for how to access dev server; http://scrape.ocdskingfisher.opendataservices.coop and http://scrape.ocdskingfisher-dev.default.opendataservices.uk0.bigv.io need to be listed.

File: docs/index.rst [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
+Kingfisher comprises three tools, which integrate to form an OCDS data pipeline:
+
+- `kingfisher-scrape <https://github.com/open-contracting/kingfisher-scrape>`_ , for downloading data from public sources of OCDS data
+- `kingfisher-process <https://github.com/open-contracting/kingfisher-process>`_, for storing OCDS data, and performing operations on it, including transformation and validation. 
+- A suite of queries that can be used to meaningfully interpret OCDS data sets
+
+Hosted Kingfisher
+=================
+
+Kingfisher has been developed primarily for internal use by Open Contracting Partnership, and OCP maintains a hosted instance. If you work or OCP or are part of the OCDS Team, see `Hosted Kingfisher <https://ocdskingfisher.readthedocs.io/en/add-overview-dos/hosted.html>`_ for details of the server, and how to access it.
```
Comment (odscjames) [inline docs/index.rst]: This path is wrong. It should just be a local reference, no?

File: docs/hosted.rst (hunk 1 above)
Comment (robredpath) [reply docs/hosted.rst]: I've removed the specifics from the doc. I figure that if we're involving people in development, we'll tell them where the dev servers are, right?

File: docs/index.rst (hunk 2 above)
Comment (robredpath) [reply docs/index.rst]: Good spot. Updated.

**Comment by robredpath** [comment]
Thanks. @odscjames. I see what you mean; I've removed scrape.rst and process.rst, and added a link to old.rst from this text.

**Pull request merged**